[Static Runtime] Move PrepackWeights to internal-only graph passes

mikeiovine opened the pull request:
Summary:
The pass introduces an `fb::` operator and thus cannot be used in OSS.

The test failure was not exposed because the Static Runtime tests have been disabled in OSS for a while. The Dev Infra folks encountered this failure when re-enabling the tests.

Test Plan: Existing tests

Differential Revision: D40724547

huydhn reviewed (approved):
LGTM!

huydhn commented:
The failure with onnx is unrelated to this change, you can either force merge this or rebase the PR to master (rebase -b master) to include the fix in https://github.com/pytorch/pytorch/pull/87803

huydhn commented:
@pytorchbot rebase

Commit:
[Static Runtime] Move PrepackWeights to internal-only graph passes (#87799)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/87799

The pass introduces an `fb::` operator and thus cannot be used in OSS.

The test failure was not exposed because the Static Runtime tests have been disabled in OSS for a while. The Dev Infra folks encountered this failure when re-enabling the tests.

Test Plan: Existing tests

Reviewed By: huydhn

Differential Revision: D40724547

fbshipit-source-id: 4c34b61921e36963b08c39c77436621666a53f03
